.github/workflows/build.yml

```diff
@@ -18,8 +13,8 @@ jobs:
 
       - name: 'Install Dependencies'
         id: install
-        run: yarn
```

Comment:
I saw a package-lock rather than a yarn.lock file, so I updated this accordingly